R/dvhydrograph-render.R

```diff
@@ -122,13 +122,16 @@ createDVHydrographPlot <- function(reportObject){
   # Add space to the top of the Y Axis
   plot_object <- RescaleYTop(plot_object)
 
+  #Add approval explanation label to the top of the plot
+  plot_object <- mtext(plot_object, text = "Displayed approval level(s) are from the source TS that statistics are derived from.", side=3, cex=0.85, line=0.33, adj=1, axes=FALSE)
+
   #Legend
   #If the time period is greater than 1 year additional x-axis labels are added so we must move the legend down further
   #Legend offset also needs to be calculated based on the number of lines and columns to be consistent in position
   legend_items <- plot_object$legend$legend.auto$legend
   ncol <- ifelse(length(legend_items) > 3, 2, 1)
   leg_lines <- ifelse(ncol==2, ceiling((length(legend_items) - 6)/2), length(legend_items))
-  legend_offset <- ifelse(ncol==2, 0.09+(0.025*leg_lines), (0.025*leg_lines))
+  legend_offset <- ifelse(ncol==2, 0.1+(0.025*leg_lines), 0.05+(0.025*leg_lines))
```

Comment:
I'm not sure if the legend_offset numbers would really fit in styles or not since there are actual calculations done to obtain them which wouldn't fit as cleanly into the CSS-like model we were using for styles.